packages/datadog-plugin-google-cloud-pubsub/test/index.spec.js

```diff
@@ -159,7 +170,7 @@ describe('Plugin', () => {
                 publish(topic, { data: Buffer.from('hello') })
               )
               .then(() => {
-                assert.strictEqual(tracer.scope().active(), firstSpan)
+                expect(tracer.scope().active()).to.equal(firstSpan)
```

Comment:
Please do not change any of these :)

Independent changes should be kept out and we actually want to use Node.js assertions.

This applies to multiple entries, not only this one, while I only use a single comment for these.